src/openrct2/interface/viewport_interaction.c

```diff
@@ -129,8 +129,13 @@ sint32 viewport_interaction_left_click(sint32 x, sint32 y)
     case VIEWPORT_INTERACTION_ITEM_SPRITE:
         switch (info.sprite->unknown.sprite_identifier) {
         case SPRITE_IDENTIFIER_VEHICLE:
-            window_ride_open_vehicle(info.vehicle);
+        {
```

Comment:
Should be indented further.

Reply:
No it should not. Same style can be found in `SawyerChunkReader` and `ObjectRepository`.

Reply:
A lot of the older code hasn't been updated to follow the current coding style. Although to be fair, this use-case wasn't included in the current [Coding Style](https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-Style) Wiki page.

Reply:
Clang format will indent braces to same level as case labels.

Reply:
@marijnvdwerf Just because SawyerChunkReader does it wrong as well, doesn't mean it's correct.